src/e2e-app/polyfills.ts

```diff
@@ -9,3 +9,16 @@ if (process.env['ENV'] === 'production') {
   Error['stackTraceLimit'] = Infinity;
   require('zone.js/dist/long-stack-trace-zone');
 }
+
+(function () {
```

Comment:
in this case i would suggest going the same way how all other polyfills that are needed for angular are used. (for example angular cli generates a polyfills.ts (https://github.com/mseemann/angular-mdl-a4-angular-cli/blob/master/src/polyfills.ts) 

also there seems to be only one browser that did not fully support CustomEvents http://caniuse.com/#feat=customevent

->
- no polyfill code in the package
- a hint in the documentation how to add the polyfill

@tb did you agree?